### PR TITLE
Fixes issue #5279 Text Editor outdents to hard left instead of going

### PR DESCRIPTION
**Pull request by mkrueger**
back an indent level

### CODE REVIEW

File: main/src/core/MonoDevelop.TextEditor.Tests/MonoDevelop.TextEditor.Tests.csproj [hunk 1]
```diff
@@ -70,7 +70,6 @@
     <ProjectReference Include="..\..\addins\MonoDevelop.SourceEditor2\MonoDevelop.SourceEditor.csproj">
       <Project>{F8F92AA4-A376-4679-A9D4-60E7B7FBF477}</Project>
       <Name>MonoDevelop.SourceEditor</Name>
-      <Private>False</Private>
```
Comment (Therzok) [inline main/src/core/MonoDevelop.TextEditor.Tests/MonoDevelop.TextEditor.Tests.csproj]: Why remove this?

**Comment by mkrueger** [comment]
Note: With local copy flag the tests don't run in the editor the source editor dll and it's deps need to be copied to the test directory. So if it's not breaking on the build bot it's a change by purpose for beeing able to run tests inside monodevelop properly.

File: main/src/core/MonoDevelop.TextEditor.Tests/MonoDevelop.TextEditor.Tests.csproj (hunk 1 above)
Comment (mkrueger) [reply main/src/core/MonoDevelop.TextEditor.Tests/MonoDevelop.TextEditor.Tests.csproj]: With local copy flag the tests don't run in the editor the source editor dll and it's deps need to be copied to the test directory. So if it's not breaking on the build bot it's a change by purpose for beeing able to run tests inside monodevelop properly.

Comment (Therzok) [reply main/src/core/MonoDevelop.TextEditor.Tests/MonoDevelop.TextEditor.Tests.csproj]: The thing is, GuiUnit resolves all the deps from the actual mono-addins initialization.

If we need a local copy, the addin is busted (no right dependencies/runtime imports) and we're only loading the right deps by chance

**Comment by mkrueger** [comment]
Y I see the build bot has problems - but in any case writing tests this way is not really possible so the local copy flag needs to be set/unset every time.

**Pull request merged**